reference/conanfile/methods/package_id.rst

```diff
@@ -65,7 +65,7 @@ If a package is building an executable to be used as a tool, and only 1 executab
         del self.info.settings.compiler
         del self.info.settings.build_type
 
-Note that this doesn't mean that the ``compiler`` and ``build_type`` should be removed for every application executable. For other things that tools, but final products to release, the most common situation is that maintaining the different builds for the different compilers, compiler versons, build types, etc. is the best approach.
+Note that this doesn't mean that the ``compiler`` and ``build_type`` should be removed for every application executable. For other things that are not tools, but final products to release, the most common situation is that maintaining the different builds for the different compilers, compiler versions, build types, etc. is the best approach.
```

Comment:
I didn't quite get what you meant by this sentence, this is my best guess. Let me know if I got it wrong :)

Reply:
No, it is good, thanks!